daq/host.py

```diff
@@ -459,6 +459,12 @@ def _docker_test(self, test_name):
             params['local_ip'] = ext_loip % self.test_port
             params['switch_ip'] = self.config['ext_addr']
             params['switch_port'] = str(self.target_port)
+
+        if 'switch_model' in self.config:
+            params['switch_model'] = self.config['switch_model']
+        else:
+            params['switch_model'] = "ALLIED_TELESIS_X230"
```

Comment:
let's not play favorites here (no default).  as above comment, this should just be part of the previous block.

Reply:
I left this fallback value for the initial commit to maintain legacy installations using AT switches so existing working system.conf files could remain unchanged.  I will remove the default value if we want to force this new required value.  Probably should document the breaking change somewhere to indicate this.

Reply:
This should be resolved now